build.sh

```diff
@@ -210,6 +212,30 @@ cp tmp/lib/*/libuuid.so.1.3.0 rootfs/lib/libuuid.so.1
 # zlib1g components
 cp tmp/lib/*/libz.so.1  rootfs/lib/libz.so.1
 
+# wpa_supplicant components
+cp tmp/sbin/wpa_supplicant rootfs/sbin/wpa_supplicant
+cp -r tmp/etc/wpa_supplicant rootfs/etc/wpa_supplicant
+
+# firmware realtek
+cp tmp/lib/modules/*/kernel/drivers/net/wireless/rtl8192cu/8192cu.ko rootfs/drivers/wireless/8192cu.ko
```

Comment:
Why place it in `rootfs/drivers/wireless/` and not in `/lib/modules/$(uname -r)/kernel/drivers/net/wireless/rtl8192cu/`?